src/features/MentorPage/MentorChips/MentorChips.tsx

```diff
@@ -0,0 +1,59 @@
+import { ChipProps } from '../../../components/Chip/types';
+import styled from 'styled-components';
+import Chip from '../../../components/Chip';
+import ShowMoreChips from './ShowMoreChips';
+
+/**
+ * Logic to show skill chips. Array of SkillPillItems
+ * should be passed to SkillPill, where styles and layout
+ * will be added. Should call MentorCards when chips are
+ * selected?
+ */
+
+const MentorSkillChips = ({ items }: { items: Array<ChipProps> }) => {
+  items.map(item => {
+    item.text = item.text.charAt(0).toUpperCase() + item.text.slice(1);
+  });
+  items.sort((a, b) => a.text.localeCompare(b.text));
+  return (
+    <ChipContainer>
+      <SkillChips id="skill-chips">
+        {items.map(item => (
```

Comment:
Here is a code suggestion for MentorChips and ShowMoreChips
I added a pic, because I was not able to indent this properly with this markdown - sorry


<img width="1680" alt="Screenshot 2022-07-28 at 11 08 42" src="https://user-images.githubusercontent.com/34128180/181456466-8633cb6b-1e8f-4990-8e98-fae266ec1838.png">

Reply:
My belief is that this code is more easy to understand than the css-classname-change. It is very apparent what happens when we click on the showmoreskills-button: we either show all skills or then we get only the first 16 items.
Css can be read later and is not important part for the logic of the app

Reply:
Actually, making it with css classes makes it easier to show three rows on any screen size, at least with my implementation. I'm not sure if that is what is wanted, or to show 16 items

Reply:
Yeah, you are not wrong actually! Probably the css class change is a good solution in this. This will change anyway later when we add some kind of pagination for the skills. 

The changing of the classnames can still be achieved with this style though. Just change the boolean to fix the styled className-selector. This solves the issue of querying the element: instead we pass the callback to the correct component =)
It is easier to follow the rabbit =)

Additionally in current implementation there is mutation of the _items_ variable. Instead of changing it directly, it is better to make a copy of it so it is not mutable. =P